recipe/meta.yaml

```
@@ -6,12 +6,12 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/PolusAI/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: f3041bb73ccb6390df5c2cf6d8ac7d4f5d91b75a3f31dd7a0664763d964c5138
+  url: https://github.com/sameeul/nyxus/archive/refs/heads/conda_gpu_v3.zip
```

Is this not an official release? 🤔 A fork with patches? If this is temporary, ok, but this should become a list of patches underneath, committed to the feedstock, before merge.

```suggestion
  url: https://github.com/sameeul/nyxus/archive/refs/heads/conda_gpu_v3.zip
  patches:
    0001-this-thing.patch
    0002-that-other-thing.patch
```

---

@jaimergp : this is just to try some stuff before release. We will switch to the original repo when we do the release. We have made some changes to CUDA detection and want to make sure those work before doing a release in the upstream repo.

---

also, since you are here, any idea why CUDA 12 CIs are missing cuda compiler, at least why CMake fails to find it?

---

I see. If you are only trying some things out, it would be cool to trim the CI matrix a bit. This feedstock has 144 entries. You can add a couple of `skip: true` lines in `meta.yaml` and rerender. For example, maybe there's no need to try all the Python versions and you can debug the CUDA stuff with Python 3.9 only.

---

We are also discussing dropping CUDA 11.2 in the nearish future ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/5339 ). CUDA 11.8 supports the same hardware, uses the same package structure in conda-forge, and has seen wide adoption. So may be a better CUDA 11 choice going forward

---

Ok. That make sense. I don't think in the `meta.yaml` I have any specific CUDA version choice. The feedstock renderer generates the CI matrix for me. I am just confused that why CUDA 12 runs are not finding the compilers from CMake. When I look at the packages that get installed in CUDA 11.2/11.8, I see a compiler package getting installed. But I don't see that in CUDA 12 runs. So, I am trying to understand what has changed and what I need to do in my recipe to get the compilers detected by CMake in CUDA 12 runs.

---

Ah ok. Added an example on how to skip a CUDA version below

https://github.com/conda-forge/nyxus-feedstock/pull/38#discussion_r1483621335